test/acceptance/features/bindAppToService.feature

```diff
@@ -842,3 +842,30 @@ Feature: Bind an application to a service
             """
         Then Service Binding "binding-request-secret" is ready
         And The application env var "SECRET_WORD" has value "aGVsbG8="
+
+    @negative
+    Scenario: Do not bind as env if there is no binding item collected from the service
+        Given Generic test application is running
+        When Service Binding is applied
+            """
+            apiVersion: binding.operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: $scenario_id
+            spec:
+                bindAsFiles: false
+                detectBindingResources: true
+                services:
+                  - group: ""
+                    version: v1
+                    kind: Service
+                    name: $scenario_id
```

Comment:
we did not create service earlier to bind to?

Reply:
The service is created along with the generic test application, but for clarity I can add a separate service so it is clear in the scenario that the service is present

Reply:
Done